resources/views/partials/collection-search.blade.php

```diff
@@ -1,11 +1,13 @@
 @php
     $link = get_home_url() . '/find-open-textbooks';
     $nonce = wp_create_nonce('find-oer');
-    $bkgd = (is_page_template(['views/template-oer.blade.php'])) ? 'bkgd-blue-navy' : 'bkgd-grey-light';
+    $bkgd = (is_page_template(['views/template-find-oer.blade.php'])) ? 'bkgd-blue-navy' : 'bkgd-grey-light';
```

Comment:
changing this template means we lose work done here: https://helga.bccampus.ca/wp/opentextbooks/open-textbooks-2/ 
`is_page_template` accepts an array of template values, so adding is preferable here.